draft-ietf-quic-transport.md

```diff
@@ -5503,6 +5503,12 @@ Reason Phrase:
   zero length if the sender chooses to not give details beyond the Error Code.
   This SHOULD be a UTF-8 encoded string {{!RFC3629}}.
 
+The application-specific variant of CONNECTION_CLOSE (type 0x1d) can only be
+sent using an 1-RTT packet ({{QUIC-TLS}}; section 4).  An application can close
+the connection during the handshake by sending a CONNECTION_CLOSE frame (type
+0x1c) with an error code of 0x15a ("user_canceled" alert; see {{?TLS13}}) in an
+Initial or a Handshake packet.
```

Comment:
Is this "An application that wishes to abandon a connection during the handshake [...]"?

Reply:
Definitely! Applied in c0b9c93. Thank you.